### ISSUE TITLE
If scheduler job crashes it is not marked as failed and other jobs are not executed

### ISSUE THREAD

**Issue by otahirs** [getgrav/grav#2996]
If there is an php error in a job execution the whole scheduler crashes to stack trace.  The job status is then never switched to _Failed_ and there is no way of finding something went wrong apart from that the last run timestamp is not changed.

suggestion => Run the job in a _try block_ and set the state to failed if it fails.

**Comment by rhukster** [comment]
True, this should be in a try-catch block.  Surprised it's not already.  I'll add this to the list of things to look at.  Cheers.

**Comment by stephan-strate** [comment]
@rhukster Any news about this?
